bin/stresstest

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#=
+if [ $# -eq 0 ] || [ $# -eq 1 ]
+then
+    echo "No arguments supplied"
+    echo "Usage: ./stresstest <NoOfThreads> <timeInMinutes>"
+    exit
+fi
+exec julia -t $1 --project=$(dirname $0)/../ -O3 "$0" -- $@
+=#
+import Pkg
+Pkg.instantiate()
+Pkg.add("CairoMakie")
```

Comment:
This is not ideal since it pollutes the package environment (due to the `Pkg.add("CairoMakie")`). What I would recommend instead is to create a new Julia environment (Project.toml + Manifest.toml) in the `bin` folder in which you `dev` the main package via `] dev ..` (or `] dev GPUInspector` if you want to use the latest release instead of the local development version of the package) and add CairoMakie.